azavea/raster-vision
# Eval Output Splitting, Unit Tests

## Overview

## Testing Instructions

To confirm the existence of the two evaluation files (one raster, one vector), please follow [these](https://github.com/jamesmcclain/raster-vision/pull/1#issuecomment-449420282) instructions.  Once you have done that, confirm the existence of `/tmp/XXX/eval/buildings_semantic_segmentation/eval.json` and `/tmp/XXX/eval/buildings_semantic_segmentation/vector-eval.json`.

### Checklist


Closes #642
